File: Java/Person.java
public class Person {
    private int idPerson;
    private String name;
    private String address;

    // Default constructor
    public Person() {
        this.idPerson = 0;
        this.name = "";
        this.address = "";
    }

    // Parameterized constructor
    public Person(int idPerson, String name, String address) {
        this.idPerson = idPerson;
        this.name = name;
        this.address = address;
    }

    // Setter
    public void setName(String name) {
        this.name = name;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    // Getter
    public int getIdPerson() {
        return this.idPerson;
    }

    public String getName() {
        return this.name;
    }

    public String getAddress() {
        return this.address;
    }
}

File: Cpp/Person.cpp
#pragma once
#include<iostream>
#include<string.h>
using namespace std;

class Person {
private:
    int idPerson;
    string Name;
    string Address;

public:
    // Constructor default
    Person() {
        idPerson = 0;
        Name = "";
        Address = "";
    }

    // Constructor dengan parameter
    Person(int idPerson, string Name, string Address) {
        this->idPerson = idPerson;
        this->Name = Name;
        this->Address = Address;
    }

    // Setter
    void setName(string Name) {
        this->Name = Name;
    }

    void setAddress(string Address) {
        this->Address = Address;
    }

    // Getter
    int getIdPerson() {
        return this->idPerson;
    }

    string getName() {
        return this->Name;
    }

    string getAddress() {
        return this->Address;
    }

    ~Person(){
    }
};